site/src/pages/index.js

```diff
@@ -29,10 +30,10 @@ const IndexPage = () => {
             <h1 className="center under-construction">Coming Soon...</h1>
             <h3 className="under-construction">See <a href="https://layer5.io/books/service-mesh-patterns">Service Mesh Patterns Book</a></h3>
           </div>
-          {/* <Navbar />
-        <Hero />
+          <Navbar />
```

Comment:
It should be the first component in the DOM tree.

Reply:
rightly pointed :eyes: